Cargo.lock

```diff
@@ -51,9 +51,9 @@ dependencies = [
 
 [[package]]
 name = "aes-gcm-siv"
-version = "0.10.3"
+version = "0.10.1"
```

Comment:
it's strange that patch versions of some of the rustcrypto crates was downgraded in the cargo.lock